Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node:lts@sha256:9b8eb61072127e898d8d2682f28508937fc21a853c6cb5587fa14d8b05ab4455
+
+WORKDIR /app
+ADD package.json package-lock.json /app/
+RUN npm ci --legacy-peer-deps
+ADD . /app
```

What is the reason you can not just add it once arent the lock files contained in adding `.` to app?
```suggestion
ADD . /app
RUN npm ci --legacy-peer-deps
```

---

Yeah, that works, thank you 🙏🏻